velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -74,6 +74,16 @@ static std::vector<std::string> kRangeFrameClauses = {
     "range between unbounded preceding and unbounded following",
 };
 
+/// Common set of window function frame clauses in ROWS mode, with current row,
+/// unbounded preceding, and unbounded following frame combinations.
+static std::vector<std::string> kRowFrameClauses = {
```

Comment:
Is this same as kRangeFrameClauses except for s/range/rows?

kRowFrameClauses -> kRowsFrameClauses (add 's' to Rows)

Reply:
Yes, they are identical except for range/rows but in subsequent PRs, tests with k preceding and k following frame bounds will be added to kRowsFrameClauses so separate variables are used.